src/Home.tsx

```diff
@@ -18,19 +25,24 @@ const useStyles = makeStyles(theme => ({
 
 const Home = () => {
   const classes = useStyles()
+  const a = useAxios('image/list/night-tag.json').response
+
+  // eslint-disable-next-line no-console
+  console.log('a', a)
```

Comment:
Deze is denk ik niet meer nodig toch?

Reply:
nee opzich niet, laat het even staan wil wat andere data inladen binnenkort.
En anders vergeet ik hoe t moet, hehe. 

Maar goed punt